### PR TITLE
Postfix expression evaluator in JavaScript

### PR DESCRIPTION
**Pull request by shivammahajancse**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(#5657 )

### CODE REVIEW

**Comment by shivammahajancse** [comment]
Please review sir  @HarshCasper @iamrajiv @ankitaggarwal23 @atarax665 @gat786

File: JavaScript/ds/postfixevaluator.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+ Input:
+ Enter the postfix expression(Please provide space after each operator or operand): 4 6 3 * + 6 -
+
+ Output:
+ 16
+
+ Time Complexity: O(n)
+ Space Complexity: O(n)
+ */
```
Comment (atarax665) [inline JavaScript/ds/postfixevaluator.js]: Add a new line at the EOF.

**Review by atarax665 — APPROVED** [review]
Just a minor change requested else everything looks fine,

**Comment by shivammahajancse** [comment]
Sir Changes done @atarax665

**Review by iamrajiv — APPROVED** [review]

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**